pkg/kubelet/config/config.go

```diff
@@ -167,6 +167,10 @@ func (s *podStorage) Merge(source string, change interface{}) error {
 		if len(updates.Pods) > 0 {
 			s.updates <- *updates
 		}
+		// Only add reconcile support here, because kubelet doesn't support Snapshot update now.
+		if len(reconciles.Pods) > 0 {
+			s.updates <- *reconciles
+		}
```

Comment:
Does the order that updates are sent matter? (e.g. would behavior change if reconciles were sent before deletes?)